bindings/python/src/session.cpp

```diff
@@ -57,61 +57,61 @@ namespace lt = libtorrent;
 namespace
 {
 #ifndef TORRENT_NO_DEPRECATE
-    void listen_on(lt::session& s, int min_, int max_, char const* interface, int flags)
```

Comment:
I did it multible commits, one for spaces and one for changes
here is the change: https://github.com/arvidn/libtorrent/pull/1541/commits/2dc55dffc9736449d1561a2b2c237012d153c2ff